### DISCUSSION TITLE
How to show code cell output in compiled pdf?

### DISCUSSION THREAD

**Discussion by iarayad** [jupyter-book/mystmd#547] [Q&A] [answered]
Hi all!

I am using Myst md to write a paper about a Python package, and I'd like to know how to show code cell outputs in the exported pdf. I compile the pdf running `myst build paper.md`

**Comment by iarayad** [comment, marked as answer]
I went for [reusing jupyter cell outputs](https://mystmd.org/guide/reuse-jupyter-outputs) from `.ipynb` files.
This means that I split some of my `.md` files such that text is in a `.md` file and code is in a `.ipynb`, and label all cells.
If anybody has a better suggestion, that'd be great :)

**Reply by rowanc1** [reply]
That is certainly what we would suggest -- glad it worked for you!!
